graphql_api/types/bundle_analysis/report.py

```diff
@@ -34,7 +35,40 @@ def resolve_bundle(
     name: str,
     filters: dict[str, list[str]] = {},
 ) -> Optional[BundleReport]:
-    return bundles_analysis_report.bundle(name, filters)
+    asset_types = None
+    if filters.get("reportGroups"):
+        asset_types = filters.get("reportGroups")
+
+    chunk_entry, chunk_initial = None, None
+    if filters.get("loadTypes"):
+        load_types = filters.get("loadTypes")
+
+        if BundleLoadTypes.ENTRY in load_types and (
+            BundleLoadTypes.INITIAL in load_types or BundleLoadTypes.LAZY in load_types
+        ):
+            chunk_entry = None
```

Comment:
if looking to retrieve by ENTRY or one of INITIAL/LAZY then it just cancels out.